crates/aiken-lang/src/parser.rs

```diff
@@ -941,6 +934,8 @@ pub fn expr_parser(
                 choice((just(Token::LeftParen), just(Token::NewLineLeftParen))),
                 just(Token::RightParen),
             ),
+            just(Token::ErrorTerm).rewind().ignore_then(seq_r.clone()),
+            just(Token::Todo).rewind().ignore_then(seq_r.clone()),
```

Comment:
Note: needed to allow when clauses without curly braces. For example:

```
when x is {
   _ -> error "foo"
 }
 ```

It doesn't parse otherwise as it expects block sequences to be wrapped between curly braces `{` `}`